workflow-templates/publish-go-nightly-task.yml

```
@@ -86,38 +102,59 @@ jobs:
           wget -q https://github.com/mitchellh/gon/releases/download/v0.2.3/gon_macos.zip
           unzip gon_macos.zip -d /usr/local/bin
 
+      - name: Write gon config to file
+        # gon does not allow env variables in config file (https://github.com/mitchellh/gon/issues/20)
+        run: |
+          cat > "${{ env.GON_CONFIG_PATH }}" <<EOF
+          # See: https://github.com/mitchellh/gon#configuration-file
+          source = [TODO_SOURCE_PATH]
+          bundle_id = TODO_BUNDLE_ID
```

```suggestion
          source = ["${{ env.DIST_DIR }}/${{ env.PROJECT_NAME }}_osx_${{ matrix.artifact.name }}/${{ env.PROJECT_NAME }}"]
          bundle_id = "cc.arduino.${{ env.PROJECT_NAME }}"
```
Since `DIST_DIR` and `PROJECT_NAME` are set according to the values in the taskfile, we are now able to configure the `source` field programmatically, reducing the amount of customization required to install the "template" in a given repository.

`bundle_id` is not necessarily required to follow this format, but I think it is a reasonable configuration.